### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by VladyslavKaminskyi**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```
Comment (olena-hul) [inline app/main.py:4]: Place each argument in 1 line

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,44 @@
...
+
+    def rate_service(self, rate: float) -> None:
+        self.average_rating = round((self.average_rating
+                                     * self.count_of_ratings
+                                     + rate)
+                                    / (self.count_of_ratings + 1),
+                                    1)
+        self.count_of_ratings += 1
+
+    def serve_cars(self, cars: list) -> float:
```
Comment (olena-hul) [inline app/main.py:38]:
```suggestion
    def serve_cars(self, cars: list[Car[) -> float:
```

File: app/main.py (hunk 1 above)
Comment (olena-hul) [reply app/main.py:4]: Fix in all places

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (VladyslavKaminskyi) [reply app/main.py:4]: If I place arguments in one line. Flake8 : E501 line too long

**Pull request closed without merging**